snapshot/vfs/vfs.go

```diff
@@ -54,6 +54,19 @@ func PathCmp(a, b string) int {
 	return strings.Compare(a, b)
 }
 
+// IsEntryBelow returns true when the entry string is a direct child
+// of parent from a filesystem perspective.  Parent has to have a
+// trailing slash.
+func IsEntryBelow(parent, entry string) bool {
+	if !strings.HasPrefix(entry, parent) {
```

Comment:
is parent guaranteed to end with a slash ? if not this would be wrong, wouldn't it be safer to prepend a check to ensure it ends with a slash and append one if not the case ?